src/components/DeFi/components/AssetInput.tsx

```diff
@@ -191,12 +199,14 @@ export const AssetInput: React.FC<AssetInputProps> = ({
         </Stack>
       )}
       {handleAccountIdChange && assetId && (
-        <AccountDropdown
-          {...(accountId ? { defaultAccountId: accountId } : {})}
-          assetId={assetId}
-          onChange={handleAccountIdChange}
-          buttonProps={{ variant: 'ghost', width: 'full', padding: 0 }}
-        />
+        <Box px={2} my={2}>
```

Comment:
We should prob put this `<Box>` inside the `AccountDropdown` for when it returns null. Then add box props so we can override the padding. This should help with that ghost padding when there is no account dropdown showing.

Reply:
For sure. Fixed up in https://github.com/shapeshift/web/pull/2789/commits/25933157c2346e572568737e84399dec2d48a487, good observation.